kube/k8s-proxy/conf/conf.go

```diff
@@ -53,6 +53,7 @@ type ConfigV1Alpha1 struct {
 	LogLevel      *string        `json:",omitempty"` // "debug", "info". Defaults to "info".
 	App           *string        `json:",omitempty"` // e.g. kubetypes.AppProxyGroupKubeAPIServer
 	KubeAPIServer *KubeAPIServer `json:",omitempty"` // Config specific to the API Server proxy.
+	ControlURL    *string        `json:",omitempty"` // URL of the Tailscale coordination server.
```

Comment:
I think whenever there's overlap with `ipn.ConfigVAlpha` we should make sure we are consistent with it, so this should probably be `ServerURL` instead.